css/styles.css

```diff
@@ -0,0 +1,197 @@
+/* stylelint-disable */
+* {
+    margin: 0;
+    padding: 0;
+    text-decoration: none;
+    text-align: center;
+  }
+ 
+.body h1 {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+  
+.menu {
+    display: flex;
+    gap: 30px;
+    justify-content: center;
+    list-style: none;
+    margin: 20px;
+  }
+  
+  .h-menu {
+    display: none;
+  }
+  
+.body h2 {
+    color: #30a02c;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 10px;
+  }
+  
+.body p {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: 40px;
+  }
+  
+  .ulist {
+    margin-top: 40px;
+  }
+  
+  .ulist ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+  }
+  
+  .ulist a {
+    margin: 10px;
+    color: #505f79;
+    font-size: 20px;
+  }
+  
+  .ulist a:hover {
+    cursor: pointer;
+    color: blue;
+  }
+  
+  @media (max-width: 770px) {
```

Comment:
- [Optional] Consider building your app mobile-first approach. This means that all your styling for mobile screens should be your default stylings and you will later add stylings for the desktop version in a media query.